python/lsst/pipe/base/butlerQuantumContext.py

```diff
@@ -91,8 +91,7 @@ def get(self, dataset: typing.Union[InputQuantizedConnection,
 
         Parameters
         ----------
-        dataset : `InputQuantizedConnection` or `list` [`~lsst.daf.butler.DatasetRef`]
-                or `~lsst.daf.butler.DatasetRef`
+        dataset
```

Comment:
I'm not sure dropping the types is the best thing here and below.

Reply:
This was recommended by @jonathansick on `#dm-docs`, on the grounds that the original form [was giving a formatting error](https://lsstc.slack.com/archives/C2B6DQBAL/p1575660845102500) and that the types are explicitly listed in the description anyway.